README.md

```diff
@@ -28,6 +29,14 @@ Astral is built with Angular Elements. You can use it in your website in under 3
   initializeAstral();
 </script>
 ```
+Optionally we can choose which widgets should appear by passing an object inside of function call:
+```html
+<script>
+  initializeAstral({
+    filterWidget: ["Contrast", "Bigger Text", "Screen Mask"]
```

Comment:
can we call this `enabledFeatures` instead? Also, we should hold a table in this section on the list of features that can be enabled.